src/Core/Application/Security/ProviderConfiguration/OpenId/UseCase/FindOpenIdConfiguration/FindOpenIdConfigurationResponse.php

```diff
@@ -104,4 +106,41 @@ class FindOpenIdConfigurationResponse
      * @var boolean
      */
     public bool $verifyPeer;
+
+    /**
+     * @var boolean
+     */
+    public bool $isAutoImportEnabled;
```

Comment:
should we set a default value ?

Reply:
It is set to false in the install / upgrade script  and should always be present, but i can also set it to false by default here to be consistent